leader-election-api/src/main/java/com/palantir/leader/PingableLeader.java

```diff
@@ -43,6 +55,7 @@ public interface PingableLeader {
     @Produces(MediaType.TEXT_PLAIN)
     @SuppressWarnings("checkstyle:AbbreviationAsWordInName") // Avoiding API break
     @Timed
+    @Handle(method = HttpMethod.GET, path = "/leader/uuid", produces = TextPlainSerializer.class)
```

Comment:
Is there no equivalent class-level annotation so we don't need to repeat the prefix for every endpoint we're annotation?

Reply:
I don't think so, the docs don't list any https://github.com/palantir/conjure-java/tree/develop/conjure-undertow-annotations

Annotations folder doesn't have anything that looks like it will do this https://github.com/palantir/conjure-java/tree/develop/conjure-undertow-annotations/src/main/java/com/palantir/conjure/java/undertow/annotations

Handle from available options javadoc seems to support endpoint only https://github.com/palantir/conjure-java/blob/develop/conjure-undertow-annotations/src/main/java/com/palantir/conjure/java/undertow/annotations/Handle.java

I could ask on the internal forum but given some endpoints were migrated like this in the past (with authors and reviewers from said internal team) it's unlikely to be a thing

Agreed that it's cumbersome